src/picker.rs

```diff
@@ -266,17 +264,129 @@ fn iterm2_from_env() -> Option<ProtocolType> {
     None
 }
 
-#[cfg(feature = "rustix")]
-fn query_stdio_capabilities(is_tmux: bool) -> Result<(Option<ProtocolType>, Option<FontSize>)> {
-    use rustix::termios::{LocalModes, OptionalActions};
+#[cfg(not(windows))]
+static ORIGINAL_IN_MODE: std::sync::Mutex<Option<rustix::termios::Termios>> =
+    std::sync::Mutex::new(None);
+
+#[cfg(not(windows))]
+fn enable_raw_mode() -> Result<()> {
```

Comment:
There's another issue that needs to be addressed here, when enable_raw_mode is called, if the code behind it exits early with an error, it may cause disable_raw_mode not to be called. Here we need a scope guard or something like that.

Reply:
Oh I didn't think of that! I will try to fix that somehow.